docs/herramientas/charlas-y-presentaciones.md

```diff
@@ -0,0 +1,122 @@
+# **Presentaciones y vídeos**
+
+En esta sección encontrarás presentaciones y vídeos de charlas impartidas por nosotros.
+
+___
+
+![](/images/pre-practica.jpg)
```

Comment:
@Edwinjrc  you need to upload the images to `static/img` and set the correct paths